main.go

```diff
@@ -33,6 +34,9 @@ func main() {
 	cfg := config.DefaultConfig()
 	rootCmd, globalFlagValues := rootcmd.New(&cfg.Hazelcast)
 	programArgs := os.Args[1:]
+	if !config.IsConfigExists() {
+		connectioncmd.New().Execute()
+	}
```

Comment:
Currently, `connection` is executed whenever there is no configuration file existed. We can change this behavior to `!config.IsConfigExists() && isInterative` based on your feedback. I guess that makes more sense since following `updateConfigWithFlags()` function already creates a default config file for a non-interactive startup.